components/calendar/src/hebrew.rs

```diff
@@ -309,64 +279,22 @@ impl Hebrew {
 
     // Converts a Biblical Hebrew Date to a Civil Hebrew Date
     fn biblical_to_civil_date(biblical_date: BookHebrew) -> HebrewDateInner {
-        let biblical_month = biblical_date.month;
-        let biblical_year = biblical_date.year;
-        let mut civil_month;
-        civil_month = (biblical_month + 6) % 12;
+        let (y, m, d) = biblical_date.to_civil_date();
 
-        if civil_month == 0 {
-            civil_month = 12;
-        }
-
-        if Self::is_leap_year(biblical_year) && biblical_month < TISHRI {
-            civil_month += 1;
-        }
-
-        debug_assert!(ArithmeticDate::<Hebrew>::new_from_lunar_ordinals(
-            biblical_year,
-            civil_month,
-            biblical_date.day
-        )
-        .is_ok());
-        HebrewDateInner(ArithmeticDate::new_unchecked(
-            biblical_year,
-            civil_month,
-            biblical_date.day,
-        ))
+        debug_assert!(ArithmeticDate::<Hebrew>::new_from_lunar_ordinals(y, m, d,).is_ok());
+        HebrewDateInner(ArithmeticDate::new_unchecked(y, m, d))
     }
 
     // Converts a Civil Hebrew Date to a Biblical Hebrew Date
     fn civil_to_biblical_date(civil_date: HebrewDateInner) -> BookHebrew {
-        let civil_year = civil_date.0.year;
-        let civil_month = civil_date.0.month;
-        let mut biblical_month;
-
-        if civil_month <= 6 {
-            biblical_month = civil_month + 6; //  months 1-6 correspond to biblical months 7-12
-        } else {
-            biblical_month = civil_month - 6; //  months 7-12 correspond to biblical months 1-6
-            if Self::is_leap_year(civil_year) {
-                biblical_month -= 1
-            }
-            if biblical_month == 0 {
-                // Special case for Adar II in a leap year
-                biblical_month = 13;
-            }
-        }
-
-        BookHebrew {
-            year: civil_year,
-            month: biblical_month,
-            day: civil_date.0.day,
-        }
+        BookHebrew::from_civil_date(civil_date.0.year, civil_date.0.month, civil_date.0.day)
     }
 
-    // ADAR = 12, ADARII = 13
     fn last_month_of_civil_hebrew_year(civil_year: i32) -> u8 {
         if Self::is_leap_year(civil_year) {
-            ADARII
+            13 // there are 13 months in a leap year
```

Comment:
Out of curiosity, why are the `u8` vlaues used directly here, but the similar function for the last month of the book Hebrew year use the month name constants?

Reply:
these are civil months; the names are biblical months